admin/app/components/solidus_admin/ui/checkbox_row/component.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::UI::CheckboxRow::Component < SolidusAdmin::BaseComponent
```

Comment:
Ditto. The template currently just works with `permission_set_ids`.

Reply:
fixed!